<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>768</width>
    <height>702</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::StrongFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Setup</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>396</x>
     <y>22</y>
     <width>353</width>
     <height>199</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="title">
    <string>AD</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>22</x>
     <y>22</y>
     <width>342</width>
     <height>155</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="title">
    <string>ORACLE</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>17</x>
     <y>377</y>
     <width>353</width>
     <height>89</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="title">
    <string>RRHH.csv file location</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnFind">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>432</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Find</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="linCSV_FileLocation">
   <property name="geometry">
    <rect>
     <x>28</x>
     <y>399</y>
     <width>331</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCancel">
   <property name="geometry">
    <rect>
     <x>525</x>
     <y>658</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnOK">
   <property name="geometry">
    <rect>
     <x>644</x>
     <y>658</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>407</x>
     <y>45</y>
     <width>331</width>
     <height>166</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="4" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Search base</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>User Name</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="linAD_UserName"/>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="linAD_SearchBase"/>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="linAD_Pwd">
      <property name="inputMethodHints">
       <set>Qt::ImhHiddenText|Qt::ImhNoAutoUppercase|Qt::ImhNoPredictiveText|Qt::ImhSensitiveData</set>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="linAD_Server"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Password</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Server</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="linAD_Port"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_14">
      <property name="text">
       <string>Port</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLineEdit" name="linORA_Pwd">
   <property name="geometry">
    <rect>
     <x>104</x>
     <y>133</y>
     <width>246</width>
     <height>24</height>
    </rect>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhHiddenText|Qt::ImhNoAutoUppercase|Qt::ImhNoPredictiveText|Qt::ImhSensitiveData</set>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>34</x>
     <y>105</y>
     <width>63</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>User Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>34</x>
     <y>133</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Password</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="linORA_SID">
   <property name="geometry">
    <rect>
     <x>104</x>
     <y>77</y>
     <width>246</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>34</x>
     <y>77</y>
     <width>20</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>SID</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="linORA_Host">
   <property name="geometry">
    <rect>
     <x>104</x>
     <y>49</y>
     <width>246</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>34</x>
     <y>49</y>
     <width>25</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Host</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="linORA_UserName">
   <property name="geometry">
    <rect>
     <x>104</x>
     <y>105</y>
     <width>246</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>179</y>
     <width>342</width>
     <height>183</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="title">
    <string>Opecus</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="linOPQ_Port">
   <property name="geometry">
    <rect>
     <x>105</x>
     <y>238</y>
     <width>246</width>
     <height>24</height>
    </rect>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhPreferNumbers</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="linOPQ_Pwd">
   <property name="geometry">
    <rect>
     <x>105</x>
     <y>322</y>
     <width>246</width>
     <height>24</height>
    </rect>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhHiddenText|Qt::ImhNoAutoUppercase|Qt::ImhNoPredictiveText|Qt::ImhSensitiveData</set>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="linOPQ_UserName">
   <property name="geometry">
    <rect>
     <x>105</x>
     <y>294</y>
     <width>246</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="linOPQ_Host">
   <property name="geometry">
    <rect>
     <x>105</x>
     <y>210</y>
     <width>246</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="linOPQ_Database">
   <property name="geometry">
    <rect>
     <x>105</x>
     <y>266</y>
     <width>246</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtEmailTo">
   <property name="geometry">
    <rect>
     <x>504</x>
     <y>318</y>
     <width>232</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_5">
   <property name="geometry">
    <rect>
     <x>399</x>
     <y>227</y>
     <width>353</width>
     <height>134</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="title">
    <string>eMail (for Export Reports)</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtEmailFrom">
   <property name="geometry">
    <rect>
     <x>504</x>
     <y>256</y>
     <width>232</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtEmailPwd">
   <property name="geometry">
    <rect>
     <x>504</x>
     <y>287</y>
     <width>232</width>
     <height>24</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>407</x>
     <y>256</y>
     <width>92</width>
     <height>85</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QLabel" name="label_15">
      <property name="font">
       <font>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>From</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_17">
      <property name="font">
       <font>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Application Pwd</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_16">
      <property name="font">
       <font>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>To</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>36</x>
     <y>211</y>
     <width>65</width>
     <height>134</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>Host</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>Port</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_11">
      <property name="text">
       <string>Database</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_12">
      <property name="text">
       <string>User Name</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_13">
      <property name="text">
       <string>Password</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLineEdit" name="linDB_FileLocation">
   <property name="geometry">
    <rect>
     <x>28</x>
     <y>499</y>
     <width>331</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnFindDB">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>532</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Find</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_6">
   <property name="geometry">
    <rect>
     <x>17</x>
     <y>478</y>
     <width>353</width>
     <height>89</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="title">
    <string>Users.db location</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_18">
   <property name="geometry">
    <rect>
     <x>415</x>
     <y>401</y>
     <width>25</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Host</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_19">
   <property name="geometry">
    <rect>
     <x>414</x>
     <y>494</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Password</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_7">
   <property name="geometry">
    <rect>
     <x>399</x>
     <y>364</y>
     <width>351</width>
     <height>274</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="title">
    <string>SFTP</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_20">
   <property name="geometry">
    <rect>
     <x>415</x>
     <y>430</y>
     <width>23</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Port</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtSFTP_UserName">
   <property name="geometry">
    <rect>
     <x>499</x>
     <y>456</y>
     <width>239</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtSFTP_Port">
   <property name="geometry">
    <rect>
     <x>499</x>
     <y>425</y>
     <width>239</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_21">
   <property name="geometry">
    <rect>
     <x>415</x>
     <y>461</y>
     <width>63</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>User Name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtSFTP_Host">
   <property name="geometry">
    <rect>
     <x>499</x>
     <y>394</y>
     <width>239</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtSFTP_Pwd">
   <property name="geometry">
    <rect>
     <x>499</x>
     <y>487</y>
     <width>239</width>
     <height>24</height>
    </rect>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhHiddenText|Qt::ImhNoAutoUppercase|Qt::ImhNoPredictiveText|Qt::ImhSensitiveData</set>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtSFTP_HostKeys">
   <property name="geometry">
    <rect>
     <x>499</x>
     <y>518</y>
     <width>239</width>
     <height>24</height>
    </rect>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhNone</set>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Normal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_22">
   <property name="geometry">
    <rect>
     <x>415</x>
     <y>523</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>HostKeys</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_23">
   <property name="geometry">
    <rect>
     <x>415</x>
     <y>558</y>
     <width>78</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Remote path</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtSFTP_RemotePath">
   <property name="geometry">
    <rect>
     <x>499</x>
     <y>553</y>
     <width>239</width>
     <height>24</height>
    </rect>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="whatsThis">
    <string/>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhNone</set>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Normal</enum>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_24">
   <property name="geometry">
    <rect>
     <x>415</x>
     <y>595</y>
     <width>78</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Filename</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtSFTP_Filename">
   <property name="geometry">
    <rect>
     <x>499</x>
     <y>588</y>
     <width>239</width>
     <height>24</height>
    </rect>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="whatsThis">
    <string/>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhNone</set>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Normal</enum>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
  <zorder>groupBox_7</zorder>
  <zorder>groupBox_6</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>groupBox_3</zorder>
  <zorder>groupBox_5</zorder>
  <zorder>groupBox</zorder>
  <zorder>groupBox_2</zorder>
  <zorder>label_5</zorder>
  <zorder>label_6</zorder>
  <zorder>label_8</zorder>
  <zorder>label_7</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>groupBox_4</zorder>
  <zorder>btnFind</zorder>
  <zorder>linCSV_FileLocation</zorder>
  <zorder>btnCancel</zorder>
  <zorder>btnOK</zorder>
  <zorder>linORA_SID</zorder>
  <zorder>linORA_Host</zorder>
  <zorder>linORA_UserName</zorder>
  <zorder>linORA_Pwd</zorder>
  <zorder>linOPQ_Port</zorder>
  <zorder>linOPQ_Pwd</zorder>
  <zorder>linOPQ_UserName</zorder>
  <zorder>linOPQ_Host</zorder>
  <zorder>linOPQ_Database</zorder>
  <zorder>txtEmailTo</zorder>
  <zorder>txtEmailFrom</zorder>
  <zorder>txtEmailPwd</zorder>
  <zorder>linDB_FileLocation</zorder>
  <zorder>btnFindDB</zorder>
  <zorder>label_18</zorder>
  <zorder>label_19</zorder>
  <zorder>label_20</zorder>
  <zorder>txtSFTP_UserName</zorder>
  <zorder>txtSFTP_Port</zorder>
  <zorder>label_21</zorder>
  <zorder>txtSFTP_Host</zorder>
  <zorder>txtSFTP_Pwd</zorder>
  <zorder>txtSFTP_HostKeys</zorder>
  <zorder>label_22</zorder>
  <zorder>label_23</zorder>
  <zorder>txtSFTP_RemotePath</zorder>
  <zorder>label_24</zorder>
  <zorder>txtSFTP_Filename</zorder>
 </widget>
 <tabstops>
  <tabstop>linORA_Host</tabstop>
  <tabstop>linORA_SID</tabstop>
  <tabstop>linORA_UserName</tabstop>
  <tabstop>linORA_Pwd</tabstop>
  <tabstop>linOPQ_Host</tabstop>
  <tabstop>linOPQ_Port</tabstop>
  <tabstop>linOPQ_Database</tabstop>
  <tabstop>linOPQ_UserName</tabstop>
  <tabstop>linOPQ_Pwd</tabstop>
  <tabstop>linCSV_FileLocation</tabstop>
  <tabstop>btnFind</tabstop>
  <tabstop>linDB_FileLocation</tabstop>
  <tabstop>btnFindDB</tabstop>
  <tabstop>linAD_Server</tabstop>
  <tabstop>linAD_Port</tabstop>
  <tabstop>linAD_UserName</tabstop>
  <tabstop>linAD_Pwd</tabstop>
  <tabstop>linAD_SearchBase</tabstop>
  <tabstop>txtEmailFrom</tabstop>
  <tabstop>txtEmailPwd</tabstop>
  <tabstop>txtEmailTo</tabstop>
  <tabstop>txtSFTP_Host</tabstop>
  <tabstop>txtSFTP_Port</tabstop>
  <tabstop>txtSFTP_UserName</tabstop>
  <tabstop>txtSFTP_Pwd</tabstop>
  <tabstop>txtSFTP_HostKeys</tabstop>
  <tabstop>txtSFTP_RemotePath</tabstop>
  <tabstop>txtSFTP_Filename</tabstop>
  <tabstop>btnOK</tabstop>
  <tabstop>btnCancel</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
